_posts/2021-02-19-status-update.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "2021-02-19: IAC Team Status Update"
+author: carabasdaniel
+categories:
+  - team
+  - status
+tags:
+  - community
+  - releases
+  - cloud-ci
+  - litmus
+  - PDKSync
+  - pdk-templates
+  - pdk
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-apache#2122`][puppetlabs-apache-pr-2122]: "(IAC-1186) Add $use_port_for_filenames parameter", thanks to [smortex][smortex]
```

Comment:
This PR was dropped off the list as it was authored by me, but it actually had contributions from two community members too and it's a customer defect:
```suggestion
- [`puppetlabs-apache#2122`][puppetlabs-apache-pr-2122]: "(IAC-1186) Add $use_port_for_filenames parameter", thanks to [smortex][smortex]
- [`puppetlabs-apache#2121`][puppetlabs-apache-pr-2121]: "(MODULES-10899) Handle PHP8 MOD package naming convention changes", thanks to [cdamage][cdamage] for the original fix, [beatchristen][beatchristen] for some enhancements for Debian 10 and [gguillotte][gguillotte] for reviewing.
```

Reply:
Here's the links required too:
```md
[beatchristen]: https://github.com/beatchristen
[cdamage]:       https://github.com/cdamage
[gguillotte]:       https://github.com/gguillotte
[puppetlabs-apache-pr-2121]: https://github.com/puppetlabs/puppetlabs-apache/pull/2124
```